### PR TITLE
Added POST-api/users Endpoint

### LINKED ISSUE #7: Endpoint: POST api/users
## User story

**Who:** **As a** developer

**What:** **I want to** be able to create a new user using a POST endpoint

**Why:** **so that we can** add a new user

## Implementation details
- Create the POST endpoint `api/users`

Look at the database diagram for details on what fields to add

### PR DESCRIPTION
**Pull request by CharlesKimMaina**
# Description
Post-api/users Endpoint

 one can create a new user using a POST endpoint

Fixes #7

# How to test
post method checked

workbench test

 `npm run dev`, in browser, https://localhost:3000/api/users 

# Checklist

- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] This PR is ready to be merged and not breaking any other functionality

### CODE REVIEW

File: src/server/api/routes/users.router.js [hunk 1]
```diff
@@ -58,4 +58,54 @@ router.get('/:id', (req, res, next) => {
...
+ *            title:
+ *              type: string
+ *            startDate:
+ *              type: string
+ *              format: date-time
+ *            endDate:
+ *              type: string
+ *              format: date-time
+ *            classId:
+ *              type: string
```
Comment (ember-b-Moss) [inline src/server/api/routes/users.router.js:90]: this schema doesn´t look correct to me. 
You can [see the schema here](https://dbdiagram.io/d/5ffc249780d742080a35cb4a).

**Review by ember-b-Moss — CHANGES_REQUESTED** [review]

File: src/server/api/routes/users.router.js (hunk 1 above)
Comment (CharlesKimMaina) [reply src/server/api/routes/users.router.js:90]: Thanks Ember,.let me correct it,

**Review by benna100 — CHANGES_REQUESTED** [review]
Have you tested this Charlie? Because there are no user tables yet right? 

Remember to link the story!

**Review by benna100 — APPROVED** [review]
Looks great Charles!

**Pull request merged**